Alexander_Putman.cfdg

```diff
@@ -0,0 +1,68 @@
+startshape CENTER
+
+rule CENTER {
+	RANDOMCIRCLE {}
+	MOVE {}
+}
+
+rule MOVE {
+	RANDOMCIRCLE {}
+	MOVE {x 4 y 2}
+}
```

Comment:
The average movement is around ~ (1, 2.5). If you were aware of some kind of RANDOM function, would you have used it here, or were you trying to create some biases in how separated things were?